RELEASE-NOTES.md

```diff
@@ -1,5 +1,17 @@
 # Wikibase DataModel release notes
 
+## Version 6.0.0 (dev)
+
+* Removed `Entity` class (deprecated since 1.0)
```

Comment:
You havev to read the patches in the correct order... This pull request includes three commits and it states in the description on which pull request it depends. Unfortunately, github doesn't handle patch chains as nice as gerrit.